src/foobar/migrations/0017_walletcorrection.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-17 15:40
+from __future__ import unicode_literals
+
+from decimal import Decimal
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import djmoney.models.fields
+import enumfields.fields
+import foobar.enums
+import uuid
+
+
+class Migration(migrations.Migration):
```

Comment:
What happens with old transactions that exist in the system?

I assume that each new transaction will generate a Correction/Log but should/can we generate an instance for the old ones?